eclipse-scout-core/src/tree/Tree.ts

```diff
@@ -300,6 +300,19 @@ export class Tree extends Widget implements TreeModel {
     ]);
   }
 
+  /**
+   * @internal
+   */
+  _postInitNodes() {
```

Comment:
- Because this is only necessary for remote case, it should be implemented in TreeAdapter.ts
- It is only necessary if autoCheckChildren is true -> Check must be outside of visitor
- Is calling updateMarkChildrenChecked necessary? Is this not done by checkNodes itself?
- Is it required to visit further after checking nodes? I think we can return false in that case which will abort visiting.
- Remove the comment or explain why it is necessary to check all children